nextflow.config

```diff
@@ -26,7 +34,7 @@ params {
     multiqc_methods_description = null
 
     // Boilerplate options
-    outdir                     = null
+    outdir                     = 'results'
```

Comment:
default `outdir` should be `null`, as we can't have a relative path on cloud system

Reply:
Thanks! I added your suggestions in the code review PR